packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx

```diff
@@ -97,6 +104,20 @@ export const useTotalsChart = () => {
     );
   }
 
+  if (accountApy) {
+    barConfig.push({
+      dataKey: 'currentApy',
+      title: <FormattedMessage defaultMessage="Current APY" />,
+      toolTipTitle: <FormattedMessage defaultMessage="Debts" />,
+      fill: 'transparent',
+      radius: [8, 8, 0, 0],
+      currencySymbol: FiatSymbols[baseCurrency]
```

Comment:
Assume that the currencySymbol does not actually have any effect here?

Reply:
Correct this was just to please the type